### PR TITLE
Python 2 __div__

### PR DESCRIPTION
**Pull request by scmc72**
As discussed in #148, I have added added the necessary components for biggus to mimic Python 2 when performing division operations.

I have redefined `__div__` so that it takes into account the dtype of arguments to 'properly' evaluate when using Python 2.

Note: when using Python 3, or `__future__` division, nothing will change as Python 3 only uses `__floordiv__` and `__truediv__`

I wasn't sure about where to put a test checking that `/` evaluates to `__div__` as the unit tests for `biggus.Array` use `__future__` division. To test this would then require a new unit test file I think?

I have added a test to check that `operator.div` evaluates properly which might be all that is needed?

Any thoughts would be much appreciated!

### CODE REVIEW

File: biggus/tests/unit/test_Array.py [hunk 1]
```diff
@@ -240,6 +235,20 @@ def test_other_no_good(self):
             a // None
 
 
+class Test___div__(unittest.TestCase, AssertElementwiseMixin):
+    def test_other_array(self):
+        a = FakeArray([2, 4])
+        b = FakeArray([2, 4])
+        r = operator.div(a, b)
+        self.assertIsInstance(r, biggus._Elementwise)
+        self.assertElementwise(r, biggus.divide(a, b))
```
Comment (pelson) [inline biggus/tests/unit/test_Array.py]: I'd be surprised if this passed on py3.

File: biggus/tests/unit/test_Array.py [hunk 2]
```diff
@@ -240,6 +235,20 @@ def test_other_no_good(self):
             a // None
 
 
+class Test___div__(unittest.TestCase, AssertElementwiseMixin):
+    def test_other_array(self):
+        a = FakeArray([2, 4])
+        b = FakeArray([2, 4])
+        r = operator.div(a, b)
```
Comment (pelson) [inline biggus/tests/unit/test_Array.py]: Looks like operator.div doesn't exist on py3.

Comment (scmc72) [inline biggus/tests/unit/test_Array.py]: Yeah - is there a way to test this in py3, or is that not possible?

Comment (pelson) [inline biggus/tests/unit/test_Array.py]: There is no special meaning for __div__ in py3k. I therefore suggest not trying to get an operator to implement these tests (both of them), and instead, just call the __div__ methods directly. (e.g. ```a.__div__(b)```)

Comment (scmc72) [inline biggus/tests/unit/test_Array.py]: Cool - I've put that in, but had change one check as `a.__div__(None)` does not raise a `TypeError` but instead returns `NotImplemented`

**Pull request merged**